### ISSUE TITLE
Feature proposal : get permission status

### ISSUE THREAD

**Issue by wynch** [evollu/react-native-fcm#849]
Hello,

this isn't really about an issue, more about a lacking feature I had to develop.

Here is my need:
- My application needs the user to accept several permissions (GPS, backgroundRefresh on iOS, Push Notifications on iOS, ....). If the user doesn't accept all the permissions, then he cannot access application and is blocked by a specific screen.

In order to develop that, I need to be able to: 
1/ check a permission
2/ activate a permission (using native popups...)

I'm using [react-native-permissions](https://github.com/yonahforst/react-native-permissions) to check / activate permissions.

The problem came when I installed react-native-fcm : both modules don't activate Notifications the same way (because of `UserNotifications` new framework introduced in iOS 10+).

**Comment by wynch** [comment]
You didn't understand me : I want to check if user accepted Notifications WITHOUT showing him the authorization popup.

**Comment by evollu** [comment]
`I want to check if user accepted Notifications WITHOUT showing him the authorization popup.`
case 1: you haven't even asked before
you can store a flag in `AsyncStorage` where you have asked for permission before or not. It can work the same way as the native code you posted.
case 2: you have asked before and user accepted/denied
calling `requestPermissions` will not show popup. You can only ask for permission once until the app gets deleted from phone.

**Comment by aatreya** [comment]
I agree that this feature would be useful. Storing a flag in AsyncStorage seems like an inelegant workaround vs. simply being able to check whether the permission is already enabled.

**Comment by evollu** [comment]
`simply being able to check whether the permission is already enabled.`
react-native-push-notification has a method to check permission. PR is welcomed for porting it over
